tests/sycl/group_functions/group_functions.hpp

```diff
@@ -53,7 +53,7 @@ using test_types = boost::mpl::list<char, unsigned int, float, double, sycl::vec
 namespace detail {
 
 template<typename T>
-using elementType = typename std::remove_reference<decltype(T{}.s0())>::type;
+using elementType = typename std::remove_reference_t<decltype(T{}.s0())>;
```

Comment:
no need for `typename` before `_t` variants, it is only required for dependent names aka if you have a `::` after a type name (ignore those after namespace names... :)), see: https://en.cppreference.com/w/cpp/keyword/typename

Reply:
good catch :+1: